app/commands/decidim/participatory_documents/admin/final_sections_publish.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryDocuments
+    module Admin
+      class FinalSectionsPublish < Rectify::Command
```

Comment:
```suggestion
      class PublishSections < Rectify::Command
```
This concept of "final" is wonky, let's be standarize a bit more, just `Publish sections` is enough.